scripts/inner/kubestellar-kube-bind

```diff
@@ -14,22 +14,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# SM_CONFIG should be set before calling this script
+# Currently this script only works "in cluster".
+# TODO: make it work for "external" users.
 
 space_name=""
 resource_to_bind=""
+kubectl_flags=()
+sub_flags=()
+
 cm_namespace=kubestellar # the namespace where the configmaps live in on the provider side
 log_dir=$(pwd)/kubestellar-kube-bind-logs
 
 while (( $# > 0 )); do
     case "$1" in
     (-h|--help)
-        echo "Usage: $0 [-X] space_name resource_to_bind"
+        echo "Usage: $0 (-h | --help | -X | \$kubectl_flag)* space_name resource_to_bind"
         echo "space_name is the name of a kube-bind consumer space, the space currently must be a direct child of the kcp 'root' workspace, note that space_name doesn't contain the 'root:' prefix"
```

Comment:
This comment is correct only for KCP, I suggest to remove it completely. We explain in the create-space that for KCP , space name is represents a workspace under root, with path "root:spacename"

Reply:
Yes, this is outdated and needs to be fixed.

Reply:
Fixed in the force-push to ebfb6293b